packages/react-instantsearch-hooks/package.json

```diff
@@ -49,7 +49,7 @@
   "dependencies": {
     "@babel/runtime": "^7.1.2",
     "algoliasearch-helper": "^3.11.1",
-    "instantsearch.js": "^4.47.0",
+    "instantsearch.js": "4.50.0",
```

Comment:
we want to leave this as caret dependency. Maybe manual intervention needed here?